### ISSUE TITLE
I `mkdir` that directory and I'm seeing the following:

### ISSUE THREAD

**Issue by zimbatm** [direnv/direnv#571]
I `mkdir` that directory and I'm seeing the following:

```shell
direnv: using nix
/home/drew/.direnvrc:33: NIXFILE: unbound variable
direnv: error exit status 1
```

I'm using this `.direnvrc` https://github.com/jwiegley/nix-config/blob/master/bin/use_nix.sh

_Originally posted by @drewboardman in https://github.com/direnv/direnv/issues/565#issuecomment-579553852_

**Comment by zimbatm** [comment]
https://github.com/jwiegley/nix-config/blob/fa8e63aaf4469ac86cc85015b5242579a8f5ad20/bin/use_nix.sh#L33
should be:
```
    if [[ -n "${NIXFILE:-}" && -f "${NIXFILE}" ]]; then
```

**Comment by drewboardman** [comment]
Yeah looks like you're using jweigley's use_nix. You can use this as an alternative for the time being:

https://gist.githubusercontent.com/mbbx6spp/731076cb8fc620b064b8e5b28fb1c796/raw/791550658f5a8a39148beaa0fe91d2d4f8cf20b0/.direnvrc
